flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -808,14 +808,8 @@ static bool updateRcvrActivityCompare(uintptr_t rcvr_id, struct rcvr_activity_fs
 			case MANUALCONTROLSETTINGS_CHANNELGROUPS_PPM:
 				group = RECEIVERACTIVITY_ACTIVEGROUP_PPM;
 				break;
-			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSMMAINPORT:
-				group = RECEIVERACTIVITY_ACTIVEGROUP_DSMMAINPORT;
-				break;
-			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSMFLEXIPORT:
-				group = RECEIVERACTIVITY_ACTIVEGROUP_DSMFLEXIPORT;
-				break;
-			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSMRCVRPORT:
-				group = RECEIVERACTIVITY_ACTIVEGROUP_DSMRCVRPORT;
+			case MANUALCONTROLSETTINGS_CHANNELGROUPS_DSM:
+				group = RECEIVERACTIVITY_ACTIVEGROUP_DSM;
```

Comment:
This was the way it was before intentionally - it allowed supporting multiple DSM inputs. for example when you have one TX for the pilot and another being relayed to a gimbal.

I'm not sure anyone is using that feature so it is probably ok to drop for now (although please anyone using it feel free to mention so). We should have a plan with the refactor to generalize having two DSM though and just calling it RECEIVERACTIVITY_ACTIVEGROUP_DSM1 and RECEIVERACTIVITY_ACTIVEGROUP_DSM2 and the HAL can take care of ordering. Sound reasonable?